visualqa/trainMLP.py

```diff
@@ -95,33 +106,81 @@ def main():
     data_root = args.dataroot
     experiment_root = args.experiment_root
 
+    # Set both the numpy and the Python random seeds.
+    random.seed(RANDOM_SEED)
+    np.random.seed(RANDOM_SEED)
```

Comment:
Great point; I checked it out. In principle it relies on the numpy seed and (I think) the TF seed, so I'll try setting that too.